src/views/TradingCompetition/contexts/CompetitionCountdownContext.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { createContext, useState, useEffect } from 'react'
+import useGetBlockCountdown from '../../../hooks/useGetBlockCountdown'
+import useGetCompetitionState from '../hooks/useGetCompetitionState'
+
+export const CompetitionCountdownContext = createContext({
+  timeUntilNextEvent: null,
+  competitionState: null,
+  isLoading: true,
+})
+
+export const CompetitionCountdownContextProvider = ({ children }) => {
+  const [timeUntilNextEvent, setTimeUntilNextEvent] = useState(null)
+  const { targetBlock, competitionState, isLoading } = useGetCompetitionState()
+  const timeUntilTargetBlock = useGetBlockCountdown(targetBlock)
```

Comment:
The `Countdown` component could bring these separate hooks in itself, rather than using context - but I expect that I _may_ need to access the combination of these two hooks (`useGetCompetitionState` & `useGetBlockCountdown `)  in other areas of the page. I can always rip context out if it turns out that I don't need to.